source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,188 @@ Install with Confluent Cloud
+            cloud environment, and find your environment ID below the environment name 
+            on the right-side panel. 
+
+            The general environment ID form is:
+
+            .. code-block::
+
+               env-ab1cde
+
+         .. step:: Find and save your cluster ID
+
+            Navigate to your :guilabel:`Cluster Overview` page and find your cluster ID 
+            on the right-side information panel. 
+
+            The general cluster ID form is:
+
+            .. code-block::
+
+               abc-1test2
+
+         .. step:: Find and save your cluster's Bootstrap server
+
+            From your :guilabel:`Cluster Overview` page, use the left-side navigation and 
+            click :guilabel:`Cluster Settings`. On the :guilabel:`Cluster Settings` page, 
+            locate your cluster's Bootstrap server.
+
+            The general Bootstrap server form is:
+
+            .. code-block::
+
+               xx.region.provider.confluent.cloud:9092
+
+         .. step:: Find and save your cluster REST endpoint
+
+            On your :guilabel:`Cluster Settings` page, find your cluster's REST 
+            endpoint.
+
+            The general REST endpoint form is:
+
+            .. code-block::
+
+               https://xxx.region.provider.confluent.cloud:443
+
+   .. step:: Add the configuration properties to your ``user.properties`` file
+
+      The location of your ``user.properties`` file varies according to your operating
+      system. For more information, see :ref:`file-location`. 
+
+      Use the following example to add the configuration properties to your 
+      ``user.properties`` file:
+
+      .. code-block::
+
+         spring.profiles.active: confluent-dev
+         migrator.confluent.environment.environment-id: <Environment ID>
+         migrator.confluent.environment.cluster-id: <Cluster ID>
+         migrator.confluent.environment.bootstrap-server: <xx.region.provider.confluent.cloud:9092>
+         migrator.confluent.environment.rest-endpoint: <https://xxx.region.provider.confluent.cloud:443>
+
+         migrator.confluent.cloud-credentials.api-key: <Cloud API key>
+         migrator.confluent.cloud-credentials.api-secret: <Cloud API Secret>
+
+         migrator.confluent.kafka-credentials.api-key: <Cluster API key>
+         migrator.confluent.kafka-credentials.api-secret: <Cluster API Secret>
+
+   .. step:: Restart Relational Migrator 
+
+      Exit Relational Migrator and relaunch it. 
+
+   .. step:: Confirm successful installation
+
+      Start Relational Migrator and in your project, navigate to the :guilabel:`Data 
+      Migration` tab. If the installation is successful, on the top right corner, a 
+      :guilabel:`Confluent Deployment model` symbol is present. 
+
+      If errors appear, check your ``log`` file. Your ``log`` file is located in an 
+      OS specific location. For more information, see :ref:`Relational Migrator File 
+      Locations <file-location>`.
```

Comment:
**Suggestion**

I also mention the Confluent errors as part of the PR your reviewing for me, you might want to link to the monitor a sync job page, once that is merged for an example of the errors:

https://github.com/mongodb/docs-relational-migrator/pull/105/files#diff-fd8cacce2301855fca6e281a317b67ac32ab2d77ad29bcb5ccd1fad506c6dfc3R160-R167

Reply:
I added the TODO here like we discussed to link out once both PR's are merged.